src/main/java/racingcar/domain/game/GameManager.java

```diff
@@ -0,0 +1,28 @@
+package racingcar.domain.game;
+
+import static racingcar.constant.GameConstant.INITIAL_START_ROUND;
+
+import racingcar.dto.Round;
+
+public class GameManager {
+    public static Integer totalRounds = 0;
```

Comment:
0으로 초기화 한다면 이렇게 바꿀 수 있겠네요 :)
```suggestion
    public static int totalRounds ;
```

Reply:
와,, 그렇네요! 사소한 부분도 체크하는 부분에서 배워갑니다... 조언 감사합니다!!